safe/definitions/hazard_classifications.py

```diff
@@ -123,7 +135,7 @@
         exposure_structure
     ]
 }
-
+# do not use these old classes until we can trace the origin
```

Comment:
Can you clarify what should we do with this class ?
Should this class be available in the wizard ? Should we use only the new one now ?